docs/test.md

```diff
@@ -0,0 +1,88 @@
+<h1>Testing</h1>
+
+The following test scenarios should be applied:
+
+## Test Scenarios
+
+<h4>Create a service</h4>
```

Comment:
Refer to one of my initial requirements about what should be mandatory and optional in these tests.

Reply:
this is to suppress the `<h4>` from the ToC - see previous comment